modules/monitoring-applying-custom-alertmanager-configuration.adoc

```diff
@@ -55,6 +55,15 @@ receivers:
 <2> `<your_matching_rules>` specifies the target alerts.
 <3> `receiver` specifies the receiver to use for the alert.
 +
+[NOTE]
+====
+Use `matchers` as the key name to indicate the matchers that an alert has to fulfill to match the node.
```

Comment:
Maybe
```suggestion
Use the `matchers` key name to specify matching rules.
```

Reply:
Updated the first part of the sentence per your suggestion but did not remove the content at the end because it changes the meaning and makes it less specific.